pkg/sql/ttl/ttljob/ttljob_test.go

```diff
@@ -317,7 +317,11 @@ func TestRowLevelTTLJobRandomEntries(t *testing.T) {
 			})
 			defer cleanupFunc()
 
+			rangeBatchSize := 1 + rng.Intn(3)
+			t.Logf("range batch size: %d", rangeBatchSize)
```

Comment:
random thought - we should have one place to randomize + log all the TTL settings

Reply:
there is more or less this above -- it's just this is the only one randomised by a cluster setting